containers-primitive/src/Map/Prim.hs

```diff
@@ -20,6 +20,8 @@ module Map.Prim
        , elems
        ) where
 
+import Control.DeepSeq (NFData (..))
```

Comment:
I see that this import is not used here. Probably it's not needed. As well as `deepseq` dependency for the `containers-primitive` package.